tasks/scheduler/replicated/replicated.yml

```diff
@@ -0,0 +1,266 @@
+###
+### Stop the Replicated and Replicated-UI containers so we can install or upgrade them
+###
+
+## TODO: add proxy support
+## TODO: add ipv6 support
+## TODO: add airgap support
+
+- name: Check if Replicated service exists
+  stat: path=/etc/init.d/replicated
+  register: replicated_upstart_service_status
+
+- name: Check if /etc/systemd/system/replicated.service exists
+  stat: path=/etc/systemd/system/replicated.service
+  register: replicated_systemd_service_status
+
+- name: Stop Replicated on Upstart
+  service:
+    name: "{{ item }}"
+    state: stopped
+  with_items:
+  - replicated
+  - replicated-ui
+  when: replicated_upstart_service_status.stat.exists or replicated_systemd_service_status.stat.exists
+  become: yes
+  become_method: sudo
+
+###
+### Autodiscover existing settings and re-use envvars like DAEMON_TOKEN when present
+###
+
+- include: config.yml
+
+- name: Check if /etc/default/replicated exists
+  stat: path={{ confdir }}/replicated
+  register: replicated_init_file
+  when:
+  - replicated_daemon_token is not defined
+
+- name: Discover daemon token
+  shell: |
+    . {{ confdir }}/replicated && echo $DAEMON_TOKEN
+  when:
+  - replicated_init_file.stat.exists
+  - replicated_daemon_token is not defined
+  register: existing_daemon_token
+
+- name: Create new daemon token
+  shell: |
+    head -c 128 /dev/urandom | tr -dc "a-zA-Z0-9" | fold -w 32 | head -n 1
+  register: new_daemon_token
+  when: 
+  - not replicated_init_file.stat.exists
+  - replicated_daemon_token is not defined
+
+- name: Set daemon token
+  set_fact:
+    replicated_daemon_token: '{{ existing_daemon_token.stdout | default(new_daemon_token.stdout) }}'
+  when: replicated_daemon_token is not defined
+
+###
+### Create Replicated user and install the global alias 
+###
+
+- include: user.yml
+
+- name: Create Replicated Alias
+  template:
+    src: ../templates/replicated_alias.j2
+    dest: /etc/replicated.alias
```

Comment:
There is one more file now in addition to the alias file, and the alias file contents no longer include the `replicated` command https://github.com/replicatedcom/install-scripts/blob/cb97005d08387d38fbe3afcc12da1dfdda63aa27/install_scripts/templates/common/cli-script.sh#L18